extensions/librdkafka/PublishKafka.h

```diff
@@ -291,10 +99,9 @@ class KafkaPool {
  private:
```

Comment:
I did not init connection here because the init function has flowfile as parameter (albeit currently the function does not use any flowfile attribute)